NEWS.md

```diff
@@ -24,6 +24,8 @@
 
 ### Other breaking changes
 
+-   The `simd` feature of the Rust library is removed in favor of
```

Comment:
can you add details on whether it changes something for the users? Should they change anything?

Reply:
Since we can update the NEWS file later, could you approve it for now?

It contains changes to the manifest file, and given the caching and the number of jobs triggered, I don't want to prolong this PR with only a documentation update.